lib/stripe.js

```diff
@@ -24,6 +24,9 @@ Stripe.USER_AGENT_SERIALIZED = null;
 var APP_INFO_PROPERTIES = ['name', 'version', 'url'];
 
 var exec = require('child_process').exec;
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var objectAssign = require('object-assign');
```

Comment:
I was suggesting the later, but I don't really care as long as there's some kind of cohesive strategy.

In your top example, why does "objectAssign" come after "util" and "EventEmitter" after "exec"? Is it reverse alphabetical? ;)